src/test/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<configuration>
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <file>target/test.log</file>
+    <encoder>
+      <pattern>%date %level [%thread] %logger{10} [%file:%line] %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%date %level [%thread] %logger{10} [%file:%line] %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <!-- Log everything with DEBUG to target/test.log file -->
```

Comment:
This may work but seems a bit convoluted. With logback you can provide two appender references in the root logger and then a level filter in the appender definitions.

Reply:
See https://logback.qos.ch/manual/filters.html#levelFilter

Reply:
Got it.